acceptance-tests/bosh_helpers.go

```diff
@@ -123,7 +142,7 @@ func dumpHAProxyConfig(haproxyInfo haproxyInfo) {
 
 // Takes bosh deploy ops files and vars.
 // Returns a session object and a callback to deserialise the bosh-generated vars store after session has executed
-func deployBaseManifest(opsFilesContents []string, vars map[string]interface{}) (*gexec.Session, varsStoreReader) {
+func deployBaseManifest(boshDeployment string, opsFilesContents []string, vars map[string]interface{}) (*gexec.Session, varsStoreReader) {
 	By("Deploying HAProxy")
```

Comment:
can we print the `boshDeployment` string here? maybe helps in debugging later

Reply:
Fixed